gradle.properties

```diff
@@ -10,7 +10,7 @@ stdlibEncodingVersion=1.0.7-SNAPSHOT
 stdlibIoVersion=0.5.5-SNAPSHOT
 stdlibJarraysVersion=0.9.5-SNAPSHOT
 stdlibJsonutilsVersion=1.0.5-SNAPSHOT
-stdlibMathVersion=1.0.5-SNAPSHOT
+stdlibRandomVersion=1.1.0-SNAPSHOT
```

Comment:
Why is this `1.1.0`, not `1.0.0` ?

Reply:
Discussed and updated the version to `0.9.0`

Reply:
We have to update the version in the original repo too right?

Reply:
> We have to update the version in the original repo too right?

Yes, will update.